script/install_macos_dependencies.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+brew update
+brew upgrade
+brew install \
+    lame \
+    libshout \
+    libconfig \
+    fftw \
+    librtlsdr \
+    soapysdr \
+    pulseaudio \
+    pkg-config
+brew link openssl
```

Comment:
is `openssl` a new dependency? should it be included in the install list as well?

Reply:
It isn't a new dependency and it is installed by default; it is used by libshout3 if I remember correctly, so probably a _hidden_ one.

On macOS 13 it doesn't have the link created by default, so the build fails because cannot find it (even if it's installed).
I don't know if it's a new default from macOS-13 or if it's a bug in the setup configuration of the worker VM